### ISSUE TITLE
About that the pod on the node is dispatched through volcano and then out of the cpu

### ISSUE THREAD

**Issue by shmily0000** [volcano-sh/volcano#2576]
In the vgpu node environment, the vgpu is not used for scheduling the cpu and memory through volcano. When the resources are insufficient, volcano will successfully schedule the po on the node, but it will immediately schedule the insufficient resources through k8s to display outofcopu. The pod is not displayed as pending only through volcano

volcano：1.6.0
K8s version: 1.23/1.23

Can you give me some help? Thank you

**Comment by shmily0000** [comment]
@wangyang0616 No, it's not the same problem.although I use the gpu node, which is set to the vgpu form, I do not request the related resources of the gpu, but only the cpu and memory. The actual resources are insufficient. Volcano will let it match the node, and then the K8S will be directly re scheduled to outofcopu

**Comment by william-wang** [comment]
@shmily0000 please provide the job yarml file and volcano-scheduler-cofigmap so that we can have a reproduce:)

**Comment by wangyang0616** [comment]
Retain the current issue.
